[Devtools] CNCF Cleaner Tasks (#258)
devfile/registry: README.md

[1]: https://img.shields.io/badge/renovate-enabled-brightgreen?logo=renovate
[2]: https://renovatebot.com

## Code of Conduct

Please reference our [Code of Conduct](CODE_OF_CONDUCT.md) for more information.

## Governance

Please reference [GOVERNANCE.md](GOVERNANCE.md) for more information.

## Users

A list of users that use this project can be found by referencing [USERS.md](USERS.md)

## Meetings

Information regarding meetings for the Devfile project can be found in [GOVERNANCE.md](GOVERNANCE.md#meetings)

## Slack

> There is a Slack channel open for discussion as part of the Kubernetes Slack, you can find it [here](https://kubernetes.slack.com/messages/devfile)

Reviewer: ```suggestion
There is a Slack channel open for discussion under the Kubernetes Slack workspace, you can find it [here](https://kubernetes.slack.com/messages/devfile)
```
